### PR TITLE
Issue 2/homepage template

### LINKED ISSUE #2: Issue 2 : Home page template
1. Create Trick Entity and controller
1. Create HP Twig base page with provided wireframes
2. Create Homepage twig template
4. Adjust responsive version

Estimate Time : 3h
Real Time :

### LINKED ISSUE #3: Issue 3 : Render tricks
5. Create getAllTricks method
6. Create renderAllTricks() in TrickController
7. Adjust Twig template with dynamic values

Estimate Time : 7h to 10h
Real Time :

### PR DESCRIPTION
**Pull request by ThomasLdev**
close #2 
close #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
         "@symfony/stimulus-bridge": "^2.0.0",
         "@symfony/webpack-encore": "^1.0.0",
         "boostrap": "^2.0.0",
+        "bootstrap": "^4.6.0",
```
Comment (crazymad) [inline package.json:6]: boostrap and bootstrap

File: src/Controller/TrickController.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+use Symfony\Component\Routing\Annotation\Route;
+
+
+class TrickController extends AbstractController
+{
+
+    /**
+     * @Route("/", name="home", methods={"GET"})
+     * @param TrickRepository $trickRepository
+     * @return Response
```
Comment (crazymad) [inline src/Controller/TrickController.php:20]: @param and @ not needed with PHP 7 +

File: src/Entity/Trick.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $category;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $created_at;
```
Comment (crazymad) [inline src/Entity/Trick.php:38]: camelCase

File: src/Entity/Trick.php [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $updated_at;
```
Comment (crazymad) [inline src/Entity/Trick.php:43]: camelCase

**Review by crazymad — CHANGES_REQUESTED** [review]

**Comment by ThomasLdev** [comment]
corrected in next issue

File: .env [hunk 5]
```diff
@@ -17,3 +17,15 @@
...
 APP_SECRET=730671a0df2a4f56e671be0c451bac7c
 ###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+#
+# DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
+
+DATABASE_URL="mysql://root:@127.0.0.1:3306/snow_tricks?serverVersion=mariadb-10.4.14"
```
[the remark this answers is not in the document]
Comment (ThomasLdev) [reply .env:27]: ok done !

File: .htaccess [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+RewriteCond %{REQUEST_FILENAME} !-d
+
+# Change 'subfolder' to be the folder you will use for your main domain.
+RewriteRule ^(.*)$ /public/$1
+
+# Change yourdomain.com to be your main domain again.
+# Change 'subfolder' to be the folder you will use for your main domain
+# followed by / then the main file for your site, index.php, index.html, etc.
+
+RewriteCond %{HTTP_HOST} ^(www.)?snow-tricks.com$
```
[the remark this answers is not in the document]
Comment (ThomasLdev) [reply .htaccess:21]: yep, deleted

File: package.json (hunk 1 above)
Comment (ThomasLdev) [reply package.json:6]: i kept the higher one

File: src/Controller/TrickController.php (hunk 2 above)
Comment (ThomasLdev) [reply src/Controller/TrickController.php:20]: phpstorm autocomplete ;) removed

File: src/Entity/Trick.php (hunk 3 above)
Comment (ThomasLdev) [reply src/Entity/Trick.php:38]: my bad, done

File: src/Entity/Trick.php (hunk 4 above)
Comment (ThomasLdev) [reply src/Entity/Trick.php:43]: same

**Review by ThomasLdev — COMMENTED** [review]
all done

**Pull request merged**